tests/transformer/test_global_local.py

```diff
@@ -0,0 +1,55 @@
+from RestrictedPython._compat import IS_PY3
+from tests import c_exec
+
+import pytest
+
+
+GLOBAL_EXAMPLE = """
+global a
+a = 1
+"""
+
+
+@pytest.mark.parametrize(*c_exec)
+def test_Global(c_exec):
+    result = c_exec(GLOBAL_EXAMPLE)
+    assert result.code is not None
+    assert result.errors == ()
+    assert result.warnings == []
+    assert result.used_names == {}
+
+
+# Example from:
+# https://www.smallsurething.com/a-quick-guide-to-nonlocal-in-python-3/
+NONLOCAL_EXAMPLE = """
+def outside():
+    msg = "Outside!"
+    def inside():
+        nonlocal msg
+        msg = "Inside!"
+        print(msg)
+    inside()
+    print(msg)
+outside()
+"""
+
+
+@pytest.mark.skipif(
+    not IS_PY3,
+    reason="Nonlocal Statement was introducted on Python 3.0 but never in Python 2")  # NOQA: E501
```

Comment:
Omitting `but never in Python 2` would allow to fit into 80 chars one line and allow to remove the `NOQA`.